CONTRIBUTING.md

```diff
@@ -95,6 +95,8 @@ test('should work', async () => {
 
 Some common test helpers, e.g. `testDir`, `isBuild` or `editFile` are available in `packages/playground/testUtils.ts`.
 
+Note: test build environment skip transpilation during tests to make it faster, This makes the js files not process by `esbuild` compared to the production environment.
+
```

Comment:
Maybe it's worth showing that Vite's test uses a different default set of Vite config?
https://github.com/vitejs/vite/blob/54e9cdd9f913ba7b650bce6657b5ff666dd0b5a8/scripts/jestPerTestSetup.ts#L102-L122

That way we cover all the unique cases when someone writes a test.